stl/inc/xmemory

```diff
@@ -1689,10 +1693,11 @@ _Alloc_ptr_t<_Alloc> _Uninitialized_move(
     // clang-format on
 
     using _Ptrval = typename _Alloc::value_type*;
-    _Seek_wrapped(
-        _Dest, _Uninitialized_move_al_unchecked(_UFirst, _ULast, _UDest, _Al,
-                   bool_constant < _Ptr_move_cat<decltype(_UFirst), _Ptrval>::_Really_trivial
-                       && _Uses_default_construct<_Alloc, _Ptrval, decltype(_STD move(*_UFirst))>::value > {}));
+    _Seek_wrapped(_Dest,
+        _Uninitialized_move_al_unchecked(_UFirst, _ULast, _UDest, _Al,
+            bool_constant < _Memcopy_is_safe<decltype(_UFirst), _Ptrval,

```

Comment:
Another statement that could really benefit from `using` (and wow, the formatting was already terrible).